test/python/scenarios.json

```diff
@@ -1,4 +1,13 @@
 {
+    "install_python_setuptools_vulnerability": {
+        "image": "debian:bookworm",
+        "features": {
+            "python": {
+                "version": "3.10",
+                "installTools": true
+            }
+        }
+    },
```

Comment:
Feel free to add this scenario and test file to https://github.com/devcontainers/features/pull/809.
It would be great to validate if we are breaking any other scenarios.